packages/terra-props-table/src/PropsTable.jsx

```diff
@@ -6,15 +6,53 @@ import Markdown from 'terra-markdown';
 
 const propTypes = {
   /**
-   * markdown file
+   * Title of component
```

Comment:
Can this be changed to just title? If that doesn't work, I think componentName would be good here as well. Component just seems more like it relates to an actual component as opposed to its name.